L77-core-cpp-third-party-identity-support-for-call-credentials.md

```diff
@@ -56,7 +56,7 @@ This doc proposes to extend the access to Google APIS directly with 3p identitie
 ```
 
 **AWS credentials**
-```
+```JSON
```

Comment:
This JSON block does not appear to be formatted correctly when viewing the file; most the text is highlighted in red.

Reply:
The comment was addressed in https://github.com/grpc/proposal/pull/221
A rogue double quote was the culprit.

Reply:
MAKE SURE TO ADD IN TOP LUXURY WEALTH AS WELL, IN THE FORM OF CODE, DIRECT OR CASH KING METHOD TO ME. AM SLOWLY BUT SURELY MAKING MY PRESCENSE KNOWN AND ITS NOT VERY LIKED SO IN PART THATS GOOD CAUSE BAD IS BETTER THEN NOTHING. I AM SMART AND BEAUTIFUL AND ALSO THE SON OF GOD AND YOU ALREADY KNOW MY DEEP ROOTS IN THAT ASPECT. MAKE AMERICA GREAT AGAIN VIA DONALD TRUMP AND MELLANIE TRUMP.  AM JUST SAYING OFF A SCRIPT THAT I ENCOUNTER IN THE BIBLE JOHN 3 16